sdk/resourcemanager/azure-resourcemanager-resources/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-management</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-core-management;current} -->
```

Comment:
This should be using the dependency tag

Reply:
Currently mgmt SDK code has a dependency on unreleased azure-core-management. After azure-core-management got released we will revert it back to dependency tag.

Reply:
It is recommended to use the `unreleased` version tag in these cases.
As mentioned [here](https://github.com/Azure/azure-sdk-for-java/blob/master/CONTRIBUTING.md#current-version-dependency-version-unreleased-dependency-version-and-released-beta-dependency-version),

>Unreleased Dependency version – Whenever possible, libraries should be using the latest released version for dependencies but there is the case where active development in one library is going to be needed by another library or libraries that are built in separate pipelines.

An unreleased entry needs to be created in version_client.txt, under the unreleased section, with the following format: `unreleased_<groupId>:<artifactId>;dependency-version`.

The dependency update tags in the pom files that required this dependency would now reference `{x-version-update;unreleased_com.azure:azure-core;dependency}`.